app/views/jst/conferences/concludedConference.handlebars

```diff
@@ -60,23 +60,29 @@
     <ul>
       {{#each recordings}}
         <li>
-          <div class="ig-row">
+          <div class="ig-row" data-id="{{recording_id}}" style="text-align: center;">
             <div class="ig-row__layout">
               <div class="ig-info">
                 <div class="ig-details">
-                  <a href="{{playback_url}}" target="_blank" class="ig-title">
+                  <a href="{{playback_url}}" target="_blank" class="ig-title"
+                    style="line-height: inherit;" data-id="{{recording_id}}" data-format="presentation">
                     {{title}}
                   </a>
-                  {{dateString created_at}}
-                  &nbsp;&#124;&nbsp;
-                  {{minutesToHM duration_minutes}}
+                  {{dateString ended_at}}
+                  &#124;&nbsp;
+                  {{durationToString duration_minutes}}
```

Comment:
can you say more about these changes?  why change from `created_at` -> `ended_at` and why change the format of the duration time string?